10up/ElasticPress
# Fix comment widget and autosuggest request

### Description of the Change

This change moves the debounce functions outside of keyup event to avoid that a new reference is created and lost the previous one. If not, the same request is sent to the server to fetch the results.

### Alternate Designs

### Benefits


### Possible Drawbacks

### Verification Process

1. Add the Elasticpress - Comments widget
2. Type e.g. "dol" (there is a delay of 500 milliseconds)
3. Check if just one request is sent to retrieve the comments.

For Autosuggest, the delay is 200 ms.

### Checklist:






### Applicable Issues



### Changelog Entry
